tests/test_lens.py

```diff
@@ -28,7 +29,9 @@
 
 from connect.governance import Governance
 
-import tempfile
+##################################################
+#################### Init ########################
+##################################################
 
 TEST_METRICS = [
```

Comment:
Can we move this to another file since we'll have other parametrizations in the near future?

Reply:
Yeah makes sense, I was debating whether to do it or not.

Reply:
😲